src/components/inputs/time-input/time-input-body.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import filterDataAttributes from '../../../utils/filter-data-attributes';
+import withMouseOverState from '../../../hocs/with-mouse-over-state';
+import { ClockIcon, CloseIcon } from '../../icons';
+import Spacings from '../../spacings';
+import styles from './time-input-body.mod.css';
+
+const getIconTheme = (isDisabled, isMouseOver) => {
+  if (isDisabled) return 'grey';
+  if (isMouseOver) return 'orange';
+  return 'black';
+};
+
+export const ClearSection = props => (
+  <div
+    onClick={props.isDisabled ? undefined : props.onClear}
```

Comment:
I've seen `null` being passed in that case. I don't mind but was wondering if there is a specific reason.

Reply:
I'm not aware of a difference and don't have any arguments for either side. I usually use `undefined` though.

If the target component was a custom component (not a "native" one like a `div`), then passing `null` would lead to the default prop not overwriting it if I'm not mistaken. Hence, I usually use `undefined`. In this case it does not matter though.